recipes/emscripten-activation/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "3.1.58" %}
+# emscripten expects specific clang version, see
+# https://github.com/emscripten-core/emscripten/blob/3.1.58/tools/shared.py#L62
+{% set clang_version = 19 %}
+
+package:
+  name: emscripten-activation
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+outputs:
+  - name: emscripten_impl_{{ cross_target_platform }}
+    script: install_emscripten_impl.sh   # [unix]
+    requirements:
+      - emscripten ={{ version }}
+      - clang {{ clang_version }}
+      - clangxx {{ clang_version }}
+    test:
+      commands:
+        - {{ CBUILD }}-emcc --version   # [unix]
+        - {{ CHOST }}-emcc --version    # [unix]
+
+  - name: emscripten_{{ cross_target_platform }}
+    script: install_emscripten.sh   # [unix]
+    run_exports:
+      strong:
+        - {{ pin_subpackage("emscripten-abi", max_pin="x.x.x") }}
+    requirements:
+      build:
+        - gettext
+      run:
+        - {{ pin_subpackage("emscripten_impl_" ~ cross_target_platform, exact=True) }}
+    test:
+      files:
+        - testfile.c
+      commands:
+        - emcc ./testfile.c
+        - node a.out.js
```

Comment:
Can you add an example for a shared library building and use?

Reply:
Fair enough. @DerThorsten, do you have a handy example we could use?

Reply:
I think the question was about actually building some concrete example of a shared library as part of the testing for the activation feedstock, rather than just `testfile.c`.

Reply:
clapack and geos are build as shared libraries:
https://github.com/emscripten-forge/recipes/tree/main/recipes/recipes_emscripten/clapack
https://github.com/emscripten-forge/recipes/tree/main/recipes/recipes_emscripten/geos